components/eamxx/src/physics/p3/impl/p3_main_impl.hpp

```diff
@@ -253,7 +254,7 @@ ::p3_main_internal(
       nr_incld, ni_incld, bm_incld, mu_c, nu, lamc, cdist, cdist1, cdistr,
       mu_r, lamr, logn0r, oqv2qi_depos_tend, precip_total_tend, nevapr, qr_evap_tend,
       ovap_liq_exchange, ovap_ice_exchange, oliq_ice_exchange,
-      pratot, prctot, hydrometeorsPresent, nk);
+      pratot, prctot, hydrometeorsPresent, nk, loc_p3constants);
```

Comment:
What does `loc_` mean?

Reply:
"local", but i see it is now not the right prefix here.